pkg/apps/apiserver/registry/deployconfig/etcd/etcd.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
This change isn't directly related to the backport, but without it I get the following build error:
```
pkg/apps/apiserver/registry/deployconfig/etcd/etcd.go:21:2: code in directory /Users/mdame/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/labels expects import "k8s.io/apimachinery/pkg/labels"
```

Reply:
Oh wow, good catch, I hope we don't have such issues any more.